tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/confirm_install.py

```diff
@@ -21,3 +21,4 @@ def tap_confirm(self):
                 *self._confirm_install_button_locator))
             Wait(self.marionette).until(expected.element_displayed(confirm))
             confirm.tap()
+            self.apps.switch_to_displayed_app()
```

Comment:
The time.sleep(1) call on line 18 doesn't make sense to me, from reading that bug, it seems to be a failure in b2g desktop, which we don't run/support anymore. I guess we could just remove it.